### PR TITLE
Feat/card

### PR DESCRIPTION
**Pull request by akdrewes**

### CODE REVIEW

File: src/app/App.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Dummy from '../images/christin-hume-505823-unsplash.jpg'
 export default function App() {
   return (
     <AppGrid>
-      <CafeCard src={Dummy} alt={`Café Image`} title={`North Coast Coffee Roastery`} address={`Deichstraße 9, Hamburg Mitte, Hamburg`} />
+      <CafeCard src={Dummy} alt={`Café Image`} title={`North Coast Coffee Roastery`} street={`Deichstraße 9`} district={`Hamburg Mitte, Hamburg`} />
```
Comment (KatharinaWuerth) [inline src/app/App.js]: Maybe you could add the zip code  - so people could search with it

Comment (KatharinaWuerth) [inline src/app/App.js]: Maybe you could also add the zip code

File: src/cards/CafeCard.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    return (
+        <CardGrid className={isHidden ? 'shorter' : ''}>
+            <Image src={src} alt={alt} />
+            <CardBackgroundColor className={isHidden ? 'shorter' : ''}/>
+            <CafeInfo>
+                <CafeTitle>{title}</CafeTitle>
+                <CafeAddress>{street}<br/>{district}</CafeAddress>
+            </CafeInfo>
+            <CafeOpeningHours>
+                <label>Öffnungszeiten</label>
```
Comment (melmare) [inline src/cards/CafeCard.js]: You could give the label an information for which this label is for.

**Review by melmare — COMMENTED** [review]
Nice code! 
Because you are working with styled-components, you could think about working with those to hide the openingHours and not to use a className.

**Pull request closed without merging**